src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -61,7 +66,7 @@ public PythonImportHandler(Interface service, ApiConfig apiConfig) {
     addImportExternal("google.gax", "path_template");
 
     // only if add enum import if there are enums
-    if (!Iterables.isEmpty(new PythonContextCommon().getEnumTypes(service.getModel()))) {
+    if (!Iterables.isEmpty(enumTable.getEnums())) {
```

Comment:
I don't think that the enumTable should be in the import handler since it does not add any extra functionality to the class. Maybe make it "global" to the snippets?